### ISSUE TITLE
feature suggestion: cache content

### ISSUE THREAD

**Issue by igneus** [YellowApple/tldrb#4]
like e.g. this python client https://github.com/lord63/tldr.py/blob/master/tldr/cli.py

Right now tldrb is quite lazy and useless offline. Content caching would make it much more useful.

**Comment by YellowApple** [comment]
I agree.  This is in progress on my end.  I'll be sure to update this issue as that progress becomes reality :)
